version/errors.go

```diff
@@ -0,0 +1,12 @@
+package version
+
+type ErrVersionMismatch struct{}
+
+func (e ErrVersionMismatch) Error() string {
+	return `
+  Some fields requested by the CLI are not available in the server schema, it seems there is a version mismatch between your CLI and the server.
+  In order to fix this issue, you might want to use the same version for both the server and the CLI.
```

Comment:
same minor version (example 0.27.3 of CLI with 0.27.2 of platform) and not the exact same full SemVer.